drivers/hmis/app/models/hmis/hoh_change_handler.rb

```diff
@@ -128,5 +128,21 @@ def self.move_in_date_not_transfered_msg(move_in_date)
     def add_warning(full_message)
       validation_errors.add(:enrollment, :informational, severity: :warning, full_message: full_message)
     end
+
+    # infer which relationship the previous HoH should have to the new HoH
+    def infer_relationship_to_new_hoh(new_hoh)
+      case new_hoh.relationship_to_ho_h
+      when 3 # Spouse
+        3 # Spouse
+      when 4 # Other relative
+        4 # Other relative
+      when 5 # Unrelated household member
+        5 # Unrelated household member
+      when 2 # Child
+        4 # Other relative a.k.a. parent. This is unexpected, because child shouldn't become HoH.
+      else
+        6 # Unrelated household member
```

Comment:
Pretty sure this should be a 5
```suggestion
        5 # Unrelated household member
```

Reply:
Oops good catch, thank you!!